cypress/support/commands.ts

```diff
@@ -70,12 +70,12 @@ export const loginViaDexUI = (username: string, password: string) => {
 Cypress.Commands.add(
   'monacoType',
   {prevSubject: true},
-  (subject, text: string) => {
+  (subject, text: string, force = true, delay = 10) => {
     return cy.wrap(subject).within(() => {
       cy.get('.monaco-editor .view-line:last')
         .click({force: true})
         .focused()
-        .type(text, {force: true, delay: 10})
+        .type(text, {force, delay})
     })
```

Comment:
Need the ability to type slower, and without force, when the specific test relies on monaco editor autoComplete.

Reply:
checked with @TCL735 as I remember him ripping something similar to these changes out recently as he chased down some flaky test patterns. he was like "totes cool yolo :shipit:" but reserved the right to be upset if it reintroduced issues. so... :+1:

Reply:
Yeah...I have worries. This pattern is used twice to test autocomplete functionality. In this particular test example (for pinned items), it has not been flaky in the branch tests.

However, for the change in `cypress/e2e/shared/explorer.test.ts`, it has been a tad flaky in OSS tests. I just extended the time from 30ms to 50ms delay between char typing @TCL735 . Hopefully that one will now behave.

Reply:
@wiedld if you want to change the `monacoType` command, then the new way shouldn't involve some arbitrary amount of delay. The changes should look something like this: https://github.com/influxdata/ui/pull/3997/files


```
Cypress.Commands.add(
  'monacoType',
  {prevSubject: true},
  (subject, text: string) => {
    return cy.wrap(subject).within(() => {
      cy.get('.monaco-editor .view-line:last').should('be.visible')
      cy.get('.monaco-editor .view-line:last').click()
      cy.get('.monaco-editor .view-line:last').type(text)
    })
  }
)
```

Reply:
I had previously refrained from merging the above mentioned PR because i didn't want to introduce so many changes to typing until we were certain that e2e tests were stable again.

e2e tests are generally stable now. But i had forgotten about this PR.

Reply:
Had an offline convo about how the tests are using the delay for use of the autocomplete feature of the monaco-editor. The use of the `{rightarrow}` works pretty well with autocomplete and a delayed typing.

However, the explorer test uses an editor cursor pause (to inject a variable) with the autocomplete closing paran. And that is more sensitive. Will change the approach on this one.